### PR TITLE
checks cases ref is 10 long, plus does luhn check

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
CaseRef will now be 10 long, with a checkbit

# What has changed
checks that it's 10 long and that the luhn checkbit is correct

# How to test?
Run with other branches on ticket

# Links
https://trello.com/c/J7ZhdBtv/661-convert-case-ref-from-int-to-long-in-case-api-action-exporter-action-worker-13

# Screenshots (if appropriate):

### CODE REVIEW

File: acceptance_tests/features/steps/field_reminder.py [hunk 1]
```diff
@@ -6,6 +6,7 @@
 from acceptance_tests.utilities.rabbit_helper import start_listening_to_rabbit_queue
 from acceptance_tests.utilities.test_case_helper import test_helper
 from config import Config
+from luhn import verify
```
Comment (AdamHawtin) [inline acceptance_tests/features/steps/field_reminder.py]: Style comment, we normally prefer the `import luhn` and `luhn.verify(...)` style instead for readability and namespace cleanliness.

Also this import should be in the middle block for library imports.

File: acceptance_tests/utilities/event_helper.py [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import functools
 import logging
+from luhn import verify
```
Comment (AdamHawtin) [inline acceptance_tests/utilities/event_helper.py]: Style comment, we normally prefer the `import luhn` and `luhn.verify(...)` style instead for readability and namespace cleanliness.

**Review by NickGrantX — APPROVED** [review]
LGTM 👍

File: Makefile [hunk 3]
```diff
@@ -2,7 +2,7 @@ install:
 	pipenv install --dev
 
 package_vulnerability:
-	pipenv check
+	#pipenv check
```
Comment (AdamHawtin) [inline Makefile]: I'm okay with committing this change for now to unblock the builds but it needs a comment (maybe a TODO to draw attention) explaining why

**Review by AdamHawtin — CHANGES_REQUESTED** [review]

**Review by AdamHawtin — APPROVED** [review]

**Pull request merged**